.github/containers/nox-cross-arch/arm64-ubuntu-20.04-python.Dockerfile

```diff
@@ -3,11 +3,13 @@
 # This Dockerfile is used to run the tests in arm64, which is not supported by
 # GitHub Actions at the moment.
 
+ARG PYTHON_VERSION="3.11"
```

Comment:
@llucax I made a mistake here defining this arg in the global scope and not consuming it within the FROM stage. I think I can defining directly within the local scope of the stage

Reply:
Fixed, but it will need a new approval

Reply:
I tested the arm64 docker file in my local environment with different python versiona and it works smoothly

Reply:
Ah, good catch. Will approve.